File: python/salario.py
h = int(input("Ingresa tus horas trabajadas: "))
he = int(input("Ingresa tus horas extras: "))

ch = h * 50 
che = he * 80 

d = ch + che
des = d * 0.05
sueldo = d - des
print(f"Trabajaste {h} horas, {he} horas extras por lo cual tu salario sería de {d} de no ser por el descuento del 5%. Tu sueldo final será de {sueldo} ")

File: c#/salario.cs
using System;

public class SueldoDeUnaPersona 
{
    public static void Main(string[] args)
    {
    double h, he, ch, che, d, des, s;
    Console.WriteLine("Ingresa tus horas trabajadas: ");
    h = Convert.ToDouble(Console.ReadLine());
    Console.WriteLine("Ingresa tus horas extras: ");
    he = Convert.ToDouble(Console.ReadLine());
    ch = h * 50.0;
    che = he * 80.0;
    d = ch + che;
    des = d * 0.005;
    s = d - des; 
    Console.WriteLine(s);
    Console.ReadKey();
    }
}

// se usa el comando "csc" para compilar y mono para ejecutar
